x-pack/legacy/plugins/epm/server/datasources/handlers.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createDatasource } from './index';
+
```

Comment:
Nit: newline here. Or is this by design?

Reply:
No. Thanks.